test/index.js

```diff
@@ -478,27 +489,39 @@ test('NSpell()', function (t) {
     st.deepEqual(
       us.suggest('dont'),
       [
-        'dent',
         'cont',
-        'font',
-        'wont',
+        'dent',
         'dint',
+        'done',
+        'dong',
+        'font',
         'dolt',
         'don',
         "don't",
         'dona',
-        'done',
-        'dong',
         'dons',
         'dost',
         'dot',
+        'wont',
         'Donn',
         'Mont',
-        'ONT'
+        'Ont'
       ],
       'should suggest alternatives including correct conjunction'
     )
 
+    st.deepEqual(
+      us.suggest('Iffect'),
+      ['Affect', 'Effect', 'Infect'],
+      'should suggest sentence-case with replaced first character'
```

Comment:
This test actually fails on main with "affect", "effect", and "Infect", as the edits were not capitalized correctly.